components/brave_wallet/browser/android_page_appearing_browsertest.cc

```diff
@@ -341,8 +341,8 @@ class AndroidPageAppearingBrowserTest : public PlatformBrowserTest {
         base::ReplaceStringPlaceholders(kPrintConsoleMarkerScript,
                                         {kConsoleMarker}, nullptr),
         content::EXECUTE_SCRIPT_DEFAULT_OPTIONS, 1);
```

Comment:
this should be `EXPECT_TRUE(ExecJs)` if we don't really use the result other than for error handling.

Such test expect will print the JS exception in case of a failure correctly.

Reply:
cc: @AlexeyBarabash

Reply:
thanks, fixed at https://github.com/brave/brave-core/pull/29767/commits/9338ef46794cbd1eb2ef3b3c07c72eb25b85132d

Reply:
fixed again after DM discussion
https://github.com/brave/brave-core/pull/29767/commits/9e0a46cdc92df176f30c4e74d1f8f165cb74cd12